W-14038153 usage reports (#206)
mulesoft/docs-general: modules/ROOT/pages/troubleshooting-usage-reports.adoc

= Troubleshooting Usage Reports

> Because platform data availability depends on activity in your organization, there might be instances in which usage reports are unavailable. 

Reviewer: ```suggestion
Because platform data availability depends on activity in your organization, in some cases, usage reports are unavailable. 
```

== Unable to Access Usage Reports in Anypoint Platform

Verify that you have have the Usage Viewer permission assigned at the root organization level. 

If you don't have the Usage Viewer permission assigned to you, contact the Anypoint Platform organization administrator. 

== A Supported Product Isn't listed

If a supported product isn't listed when you open the product list, click *Switch View*.

For more information about product availability by view, see xref:usage-reports.adoc#products-usage-reports[available products].

== Usage Reports Don't Display Data for Runtime Fabric Apps

If you don't see any data for Runtime Fabric apps, ensure that:

* You're using Runtime Fabric agent version 1.13.112, 2.1.30, or later. 
* After updating the Runtime Fabric agent, you manually redeployed each Runtime Fabric app with a supported date-patch version.

If you're still missing Runtime Fabric apps in your usage reports, contact your account executive to request a report of apps that are not upgraded and what actions you need to take to update them.

== A Month or a Day is Unavailable in the Date Picker 

> This means that data is unavailable for the selected day or month. 

Reviewer: ```suggestion
If the date picker doesn't show a month or day, data is unavailable for the selected period. 
```

== Report Says No Data

> If you select a product and daily or monthly report in new usage reports, the report returns a `No Data` message. 

Reviewer: ```suggestion
If you select a product and daily or monthly report in new usage reports, and the report returns a `No Data` message, no data is available for the selected period.
```

Reviewer: I guess, for these, you give the situation and then the reason.

I think you can combine them and avoid the "This means that..." construction.

> This means that there is no data available for the selected day or month. 

Reviewer: ```suggestion
```

== No Report Available for This Time Period Message

> After selecting a month in legacy usage reports, the report returns a `No report available for this time period` message instead of showing data.

Reviewer: ```suggestion
If, after selecting a month in legacy usage reports, the report returns `No report available for this time period`, no data is available for the selected month.
```

This means there is no data available for the selected month. 

> == Deployed Mule app Isn't Listed in the Mule Runtime Usage Report

Reviewer: ```suggestion
== Deployed Mule App Isn't Listed in the Mule Runtime Usage Report
```

> If a deployed Mule app isn't listed in the Mule Runtime usage report, ensure that you are using a supported version of Mule Runtime. For more information, see xref:mule-runtime-usage.adoc#supported-runtime-versions[supported runtime versions].

Reviewer: ```suggestion
If a deployed Mule app isn't listed in the Mule Runtime usage report, make sure you're using a supported version of Mule runtime. For more information, see xref:mule-runtime-usage.adoc#supported-runtime-versions[supported runtime versions].
```
